### PR TITLE
Sort extension secrets by key

### PR DESCRIPTION
**Pull request by jsierles**
Extension secrets should always show up in the same order. For now, that will be alphabetical.

Based on feedback from this article: https://benhoyt.com/writings/flyio-and-tigris/?moving

### CODE REVIEW

File: internal/command/extensions/core/core.go [hunk 1]
```diff
@@ -472,12 +473,19 @@ func setSecretsFromExtension(ctx context.Context, app *gql.AppData, extension *E
...
+		var keys []string
+
+		for key := range secrets {
+			keys = append(keys, key)
 		}
-	}
+		sort.Strings(keys)
 
+		for _, key := range keys {
+			fmt.Println(key + ": " + secrets[key].(string))
```
Comment (dangra) [inline internal/command/extensions/core/core.go:485]: should it use `fmt.Fprintf(io.Out, "%s: %s\n", key, secrets[key].(string))` ?

File: internal/command/extensions/core/core.go [hunk 2]
```diff
@@ -472,12 +473,19 @@ func setSecretsFromExtension(ctx context.Context, app *gql.AppData, extension *E
...
 	} else {
-		fmt.Fprintf(io.Out, "Set one or more of the following secrets on your target app.\n")
-		for key, value := range secrets {
-			fmt.Println(key + ": " + value.(string))
+		fmt.Fprintf(io.Out, "Set the following secrets on your target app.\n")
+		var keys []string
+
+		for key := range secrets {
+			keys = append(keys, key)
 		}
```
Comment (dangra) [inline internal/command/extensions/core/core.go:481]: you can replace all of that with `keys := lo.Keys(secrets)`

File: internal/command/extensions/core/core.go (hunk 1 above)
Comment (jsierles) [reply internal/command/extensions/core/core.go]: 👍

File: internal/command/extensions/core/core.go (hunk 2 above)
Comment (jsierles) [reply internal/command/extensions/core/core.go]: 🧇

**Review by dangra — APPROVED** [review]

**Pull request merged**